### DISCUSSION TITLE
Suggestions on Variable Naming

### DISCUSSION THREAD

**Discussion by ichenhua** [themeselection/flyonui#17] [Ideas / Request new feature]
FlyonUI is currently the best TailwindCSS framework I have used, and I hope the author can maintain it for a long time.

Now, I have a small suggestion.

When defining CSS variables, the system utilizes abbreviated forms, which results in a poor experience in terms of intuitiveness and user-friendliness. It is recommended that in the next update, the full spelling should be used instead of abbreviations.

For example:
--p: 62.3083% 0.188015 259.814527;

to
--primary: 62.3083% 0.188015 259.814527;

**Comment by PruthviPraj00** [comment]
Hello @ichenhua,

Thank you for your feedback!

You’ve raised a valid point. However, as mentioned, we use DaisyUI for our CSS structure. The decision to keep the variable names prefixed with `--p` stems from the fact that some variable names can be quite lengthy and hard to remember, such as `--secondary-content`. 

Using shorthand like `--sc` simplifies writing, and although it may take some time to adjust, it’s ultimately more efficient than typing out long variable names in your code.
